nmxact/xact/fs.go

```diff
@@ -155,6 +155,7 @@ func nextFsUploadReq(s sesn.Sesn, name string, data []byte, off int) (
 	}
 
 	room := min(s.MtuOut(), cMaxFSUploadChunk) - len(emptyEnc)
+	room = cMaxFSUploadChunk
```

Comment:
We should probably note that you mean to leave these overwritten initializations on purpose.

Reply:
Agreed. See the other comment below 👇 that I tagged you in as well and once we decide on that, I’ll make some more changes.